### DISCUSSION TITLE
how to add arrow from source to target or vice versa

### DISCUSSION THREAD

**Discussion by AnubhabMondal** [networkx/networkx#5256] [Q&A] [answered]
HI everyone . I am new here in Networkx. I want to show a bank statement graph where its shows money transaction graph.
Now i have data like this 

![Screenshot (125)](https://user-images.githubusercontent.com/43597328/149320556-96102d44-2ec8-4626-b947-36f7c3a2c2f8.png)

I want to show transactions like this
![Untitled Diagram drawio](https://user-images.githubusercontent.com/43597328/149321716-7e1a9055-dea7-49bc-9aff-a5ed5ab6f368.png)

like if credit:

account holder name --------[amount money]------>Descriptions

If Debit:
account holder name <------[amount money]------Descriptions

in a same graph credit and debit all transctions.
now in my code i can not use both arrow moves in a same graph . 
my code 

`import networkx as nx
import matplotlib.pyplot as plt
from tqdm import tqdm
df = pd.read_csv("first_table1.csv")

df.head(100)
target = df['Description']
source = "sanu"
relations = df['DR'].fillna(df.pop('CR'))
print(relations)
kg_df = pd.DataFrame({'source':source, 'target':target, 'edge':relations})
G=nx.from_pandas_edgelist(kg_df, "source", "target", 
                          edge_attr=True, create_using=nx.MultiDiGraph())
plt.figure(figsize=(12,12))
pos = nx.spring_layout(G, k = 0.5)
nx.draw(G, with_labels=True, node_color='skyblue',node_size=1500, edge_cmap=plt.cm.Blues, pos = pos)
nx.draw_networkx_edge_labels(G, pos=pos)
plt.show()`

Can anyone please help me on this ? That would be really be helpfull. Thanks in advance!! @rocky @lonnen

**Comment by dschult** [comment, marked as answer]
Something like:  ???
```python
df = pd.read_csv("first_table1.csv")
G=nx.Graph()
for target, cr, dr in zip(df['Description'], df['CR'], df['DR]):
    if cr.isna():   # I might have the wrong syntax for checking if a value is NA
        G.add_edge(target, source, dr)
    else:
        G.add_edge(source, target, cr)
```
(not tested)

**Reply by byteshiva** [reply]
Some correction in syntax
```
df = pd.read_csv("first_table1.csv")
G=nx.Graph()
for target, cr, dr in zip(df['Description'],  df['CR'], df['DR']):
    if pd.isna(cr):   # this checks if value is nan or not
      G.add_edge(target, source, debit=dr) 
    else:
      G.add_edge(source, target, credit=cr)
```
